src/amo/components/AddonMoreInfo.js

```diff
@@ -17,14 +17,21 @@ export class AddonMoreInfoBase extends React.Component {
   render() {
     const { addon, i18n } = this.props;
 
+    let homepage = addon.homepage;
+    if (homepage && !homepage.match(/^https?:\/\//)) {
```

Comment:
Should this trim whitespace at all? Hopefully it won't have it but would be good to check. Because ' http://...' would end up being prefixed even though within an href it would still work.